assets/javascripts/openqa.js

```diff
@@ -626,15 +626,22 @@ function renderHttpUrlAsLink(value) {
     value = String(value);
   }
   const urlRegex = /https?:\/\/[^\s,]*/g;
+  // Regex to match an array of space-separated URLs (to print one per line)
+  const arrayOfUrlsRegex = /^(https?:\/\/[^\s,]*)(\s+https?:\/\/[^\s,]*)+\s*$/g;
   let lastIndex = 0;
   let match;
+  let arrayOfUrls = arrayOfUrlsRegex.exec(value) !== null;
   while ((match = urlRegex.exec(value)) !== null) {
     if (match.index > lastIndex) {
       fragment.appendChild(document.createTextNode(value.substring(lastIndex, match.index)));
     }
     const a = document.createElement('a');
     a.href = a.textContent = match[0];
     fragment.appendChild(a);
+    if (arrayOfUrls && oneUrlPerLine) {
```

Comment:
here it would be better to use Array.isArray 

Something like, but then you should also change the variable name. (I think it is a little weird that _array_ appears so many times)) 

```suggestion
    if (Array.isArray(arrayOfUrls) && oneUrlPerLine) {
```